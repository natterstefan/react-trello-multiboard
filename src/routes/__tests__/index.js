import React from 'react'
import { shallow } from 'enzyme'

import Routes from '../'

describe('Routes', () => {
  test('should render without throwing an error', () => {
    expect(shallow(<Routes />)).toMatchSnapshot()
  })
})
