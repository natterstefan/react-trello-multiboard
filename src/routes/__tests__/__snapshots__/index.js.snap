// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render without throwing an error 1`] = `
<Connect(ConnectedRouter)
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Switch>
    <Route
      key="/privacy"
      path="/privacy"
      render={[Function]}
    />
    <Route
      key="/github"
      path="/github"
      render={[Function]}
    />
    <Route
      key="/"
      path="/"
      render={[Function]}
    />
  </Switch>
</Connect(ConnectedRouter)>
`;
